### PR TITLE
Create account page

### PR DESCRIPTION
**Pull request by dm1031**

### CODE REVIEW

File: src/server/routes/api/users.ts [hunk 1]
```diff
@@ -6,66 +6,81 @@ import Product from '../../models/product'
...
+        .then(orders => res.send(orders))
+        .catch(next)
+    }
+  }
+)
+
+route.post(
+  '/',
+  (req: express.Request, res: express.Response, next: express.NextFunction) => {
+    User.create({ name: req.body.name, password: req.body.password })
```
Comment (bailiewu) [inline src/server/routes/api/users.ts]: This should be done on the class model

**Review by bailiewu — CHANGES_REQUESTED** [review]
create user should be a model method

File: src/server/routes/api/users.ts (hunk 1 above)
Comment (dm1031) [reply src/server/routes/api/users.ts]: roger

**Review by hgranthorner — DISMISSED** [review]
looks good 2 me

**Review by bailiewu — APPROVED** [review]
Approving for now, current issue:
sending status twice when user is logging in with incorrect pw.

**Pull request merged**